Assets/Features/Building/Scripts/Grid/Grid.cs

```diff
@@ -52,6 +54,25 @@ void Start()
             UpdateTraversable();
         }
 
+        /// <summary>
+        /// Gets the utilities of a given type. This will only return the ones that have been built
+        /// </summary>
+        /// <param name="utilityType">The type of the utility</param>
+        /// <returns>A list with all the utilities</returns>
+        public List<Vector3Int> GetUtilities(UtilityType utilityType)
+        {
+            List<Vector3Int> positions = new List<Vector3Int>();
+            foreach (Vector3Int position in _utilityLocations[utilityType])
+            {
+                //check if cell is built
```

Comment:
"Check if cell is built"? Does that mean that utilities that are under construction are also added to the list?
If yes, isn't it a better idea to add them once they're build?